finetuner/client/__init__.py

```diff
@@ -0,0 +1,3 @@
+import hubble
+
+hubble.login()
```

Comment:
maybe i's better to ask user to decide login rather than call the login automatically and open the browser?